FP-Erlang/Week 3/Assignment 2/Rahul/assignment2.erl

```diff
@@ -0,0 +1,58 @@
+-module(assignment2).
+-export([qsort/1, insort/1, bubsort/1, selsort/1, mergesort/1, unique/1, replicate/2, reverse/1, zip/2, fibonacci/1]).
+
+qsort([]) -> [];
+qsort([Pivot|T]) ->
+qsort([X||X <- T, X<Pivot]) ++ [Pivot] ++ qsort([X||X <- T, X>=Pivot]).
+
+insort([H|T]) -> insort([H],T).
+insort(Sorted,[]) -> Sorted;
+insort(Sorted,[H|T]) -> insort([X||X <- Sorted, X<H]++[H]++[X||X <- Sorted, X>=H],T).
```

Comment:
Do you notice how similar insertion sort and quick sort are in functional programming?
Rather, all sorts are similar in functional languages. These names like quick sort, insertion sort are given to algorithms implemented in imperative languages like C, C++, Java